test/conformance/conformance_global_control.cpp

```diff
@@ -271,7 +375,7 @@ TEST_CASE("terminate_on_exception: enabled") {
                     if (suppress_unreachable_code_warning) {
                         throw std::exception();
                     }
-                });
+                    });
```

Comment:
I would personally prefer the previous variant, as more logically consistent

Reply:
```suggestion
                });
```